crates/iota/src/keytool.rs

```diff
@@ -213,6 +205,14 @@ pub enum KeyToolCommand {
         #[clap(long)]
         base64pk: String,
     },
+    /// Update an old alias to a new one.
+    /// If a new alias is not provided, a random one will be generated.
+    UpdateAlias {
+        old_alias: String,
```

Comment:
I want to make the suggestion (done in a separate PR of course) to turn this into `key_identity: KeyIdentity` field, so you can also provide the address for which you want to update/set a new alias.

Reply:
Please open an issue 👍🏻

Reply:
https://github.com/iotaledger/iota/issues/4889